ste/mgr-JobPartMgr.go

```diff
@@ -547,8 +547,8 @@ func (jpm *jobPartMgr) createPipelines(ctx context.Context) {
 
 	var statsAccForSip *PipelineNetworkStats = nil // we don't accumulate stats on the source info provider
 
-	// Create source info provider's pipeline for S2S copy.
-	if fromTo == common.EFromTo.BlobBlob() || fromTo == common.EFromTo.BlobFile() {
+	// Create source info provider's pipeline for S2S copy or download (in some cases).
+	if fromTo == common.EFromTo.BlobBlob() || fromTo == common.EFromTo.BlobFile() || fromTo == common.EFromTo.BlobLocal() {
```

Comment:
Historically we didn't really use the SIP at all for downloads. In fact, it has to manually be created for the downloader if we intend to use it.

This isn't a problem in FileLocal even with SMB properties because files has no OAuth support.